avm/client/src/avmHelpers.ts

```diff
@@ -162,6 +162,9 @@ export async function callAvm(
         return {
             retCode: -1,
             errorMessage: 'marine-js call failed, ' + e,
-        } as any;
+            data: new Uint8Array(),
```

Comment:
Can we make a separate result for marine-js, wrapping marine-js into AquaVM error looks wierd.

Reply:
Let's just do not handle errors on this level at all. The consumer should be responsible to handle errors in the `fn` passed into arguments